static/js/components/AnnotationsTable.jsx

```diff
@@ -75,6 +83,40 @@ const getMuiTheme = (theme) =>
     },
   });
 
+const dialogTitleStyles = (theme) => ({
+  root: {
```

Comment:
These isn't many annotations in the demo data to see how it renders when there is a lot of them. Did you try it by any chance while adding this styling? I am wondering if it doesn't overflow or anything, as it doesn't seem to be constrained in width and height.

Reply:
I just tested it with what is in there. We don't have all that many columns, and the height seems OK. I suggest we move forward like this for now and then see how it works in production.